lib/Models/Terria.js

```diff
@@ -418,7 +459,7 @@ Terria.prototype.start = function(options) {
  * @param {String} newUrl The new URL of the application.
  * @return {Promise} A promise that resolves when any new init sources specified in the URL have been loaded.
  */
-Terria.prototype.updateApplicationUrl = function(newUrl) {
+Terria.prototype.updateFromApplicationUrl = function(newUrl) {
```

Comment:
This is a breaking change.  Add a new function with the old name and call this one after calling `deprecationWarning`.  Also, look for all the places we call it.  There's at least one.